ai_bridge/static/src/components/chatter_topbar/chatter.esm.js

```diff
@@ -0,0 +1,22 @@
+/** @odoo-module **/
+
+import {registerPatch} from "@mail/model/model_core";
+
+registerPatch({
+    name: "Chatter",
+    recordMethods: {
+        async onClickAiBridge(aiBridge) {
+            if (this.isTemporary) {
+                const saved = await this.doSaveRecord();
+                if (!saved) {
+                    return;
+                }
+            }
+            await this.env.services.orm.call("ai.bridge", "execute_ai_bridge", [
```

Comment:
If it throws an error, "_execute" doesn't create the record. Should it create it independently? Would it be better to create it and then execute it? (in the model)